src/cpp/include/openvino/genai/whisper_pipeline.hpp

```diff
@@ -12,36 +12,45 @@
 #include "openvino/genai/llm_pipeline.hpp"
 #include "openvino/genai/whisper_generation_config.hpp"
 
-namespace ov::genai {
+namespace ov {
+namespace genai {
 
 using OptionalWhisperGenerationConfig = std::optional<WhisperGenerationConfig>;
 
 using RawSpeechInput = std::vector<float>;
 
 /**
- * @brief base class for chunk streamers. In order to use inherit from from this class and implement put, and methods
- *
- * @param m_tokenizer tokenizer
+ * Base class for chunk streamers. In order to use inherit from from this class and implement put, and methods
+ * ChunkStreamerBase is deprecated and will be removed in 2026.0.0 release. Use StreamerBase instead.
  */
-class OPENVINO_GENAI_EXPORTS ChunkStreamerBase : public StreamerBase {
+class OPENVINO_GENAI_EXPORTS ChunkStreamerBase {
 public:
+    OPENVINO_DEPRECATED(
```

Comment:
you can deprecated the whole class instead of deprecation of its methods

To avoid having deprecation warnings in our code (for where where it cannot be fixed), you can wrap such places with `OPENVINO_SUPPRESS_DEPRECATED_START` and `OPENVINO_SUPPRESS_DEPRECATED_END` (grep usage in the project)